content/materials.md

```diff
@@ -50,6 +50,8 @@
 
 ::: {.module-resource .slides}
 [Slides (20 minutes)](slides/data-formats-and-inspection.md)
+
+* [Extra (more content to explore on your own time!)](slides/more-slides-on-data-formats.md)
```

Comment:
What do you think about this patch?

```
diff --git a/content/materials.md b/content/materials.md
index eb66d87..380ac69 100644
--- a/content/materials.md
+++ b/content/materials.md
@@ -50,8 +50,10 @@
 
 ::: {.module-resource .slides}
 [Slides (20 minutes)](slides/data-formats-and-inspection.md)
+:::
 
-* [Extra (more content to explore on your own time!)](slides/more-slides-on-data-formats.md)
+::: {.module-resource .slides-extra}
+[Extra (more content to explore on your own time!)](slides/more-slides-on-data-formats.md)
 :::
 
 ::: {.module-resource .exercise}
diff --git a/custom.scss b/custom.scss
index f5a61be..1a102bb 100644
--- a/custom.scss
+++ b/custom.scss
@@ -50,6 +50,12 @@ ul ul, ol ol, ul ol, ol ul {
   position: absolute;
 }
 
+.slides-extra:before {
+  content: "➕";
+  left: 5px;
+  position: absolute;
+}
+
 .video:before {
   content: "🎥";
   font-family: var(--bs-body-font-family);
```

Reply:
f1a902f * trey-overview-todos Add css for extra module content

Reply:
Why didn't that commit automatically become a link? :confused: